src/main/webapp/ts/profileBuilder.ts

```diff
@@ -1,39 +1,67 @@
-type authInfoFromServlet = {
+type authInfo = {
     loginUrl: string;
     logoutUrl: string;
     hasProfile: boolean;
 };
 
 function setForm(value: string) {
-    const menteeForm =  document.getElementById('Mentee');
-    const mentorForm = document.getElementById('Mentor');
+
     const userTypeButton = document.getElementById('user-input');
+    const menteeInputs: string[] = ['name-input', 'school-input', 'major-input'];
+    const mentorInputs: string[] = ['profName-input', 'company-input', 'careerTitle-input', 'bio-input', 'specialty-input']
 
-    if(menteeForm && mentorForm && userTypeButton){
-        if(value == 'Mentee') {
-            menteeForm.setAttribute('style','display:block;');
-            mentorForm.setAttribute('style','display:none;');
-            userTypeButton.setAttribute('value','Mentee');
+    if (userTypeButton) {
+        const visibleId = value;
+        const hiddenId = visibleId == 'Mentee' ? 'Mentor' : 'Mentee';
```

Comment:
This reduces the duplicated logic - I think it can still be a little better.

```
   const hiddenInputs = hiddenId == 'Mentee' ? menteeInputs : mentorInputs;
   const visibleInputs = visibleId == 'Mentee' ? menteeInputs : mentorInputs;
```

Reply:
Implemented!